album/pokemon-scarlet-violet-super-music-collection.yaml

```diff
@@ -1894,7 +1896,7 @@ URLs:
 Referenced Tracks:
 - Mochi Mayhem
 ---
-Track: Title Screen 2
+Track: Title Screen
```

Comment:
oh my god they actually gave two songs on the same album the exact same name. nintendo why are you like this

Reply:
nintendo